packages/realm-web/src/FunctionsFactory.ts

```diff
@@ -0,0 +1,74 @@
+import { Transport } from "./transports/Transport";
+
+/**
+ * A list of names that functions cannot have to be callable through the functions proxy.
+ */
+const RESERVED_NAMES = ["inspect", "callFunction"];
+
+interface CallFunctionBody {
+    name: string;
+    arguments: any[];
+    service?: string;
+}
+
+export interface FunctionsFactoryConfiguration {
```

Comment:
I think we can remove this type and pass the needed configs as arguments

Reply:
I originally made this a config object because I don't want to repeat the arguments twice (in the constructor and the exported `create` function) - but you're right, two parameters might as well be positional arguments.

Reply:
It turns out I'll have to revert that change with my upcoming `kh/mongodb-realm-services` branch, as the functions factory now takes four parameters.